### ISSUE TITLE
FileNotFoundError thrown in GUI after updating to 2.3.1

### ISSUE THREAD

**Issue by Abrolo** [DeepLabCut/DeepLabCut#2169]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

The DLC GUI cannot create or load config files as of 2.3.1, FileNotFoundError is reported at every attempt to load projects. The terminal version still seems to work though.

Note that the program was working just fine on 2.3.0, I was using it already on a project, but after updating i can no longer use the GUI. 

### Operating System

operating system Windows 10

### DeepLabCut version

dlc version 2.3.1

### DeepLabCut mode

single animal

### Device type

cpu Processor	AMD Ryzen 7 5800H with Radeon Graphics, 3201 Mhz, 8 Core(s), 16 Logical Processor(s)

### Steps To Reproduce

1. Start the DLC GUI.
2. Load any project config file.
3. FileNotFoundError.

### Anything else?

![image](https://user-images.githubusercontent.com/45576147/223732875-54d4a4a8-19ee-4c60-b9b8-a4ff663c8f34.png)

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by jeylau** [comment]
Hi @Abrolo, I cannot reproduce this error (on macOS); I will look whether it is Windows-specific, and get back to you.

**Comment by benlansdell** [comment]
We have also seen this issue (on Windows machine)

**Comment by OmerBarkai** [comment]
Experiencing the exact same issue - following...

Thanks for the support!

**Comment by VAmoroso13** [comment]
Your details are sparse, but it sounds like the following issue:

https://forum.image.sc/t/cant-load-the-poject-due-to-file-not-found-error-in-latest-version-2-3-1/78268/3?u=vamoroso

**Comment by jeylau** [comment]
I realized that the file is not found as it was not bundled with deeplabcut when publishing it on pypi. It will be fixed very soon with #2170 😌

**Comment by Nike-Nay** [comment]
Thanks a lot for the update. I am facing the same issue since installing dlc last night, where the models.json file is missing.

**Comment by kdhenderson** [comment]
Hi, I am seeing what I believe is the same error. I installed DLC on my windows 10 machine (I'm a mac user, but want to use the GPU on my windows machine). When I try to create a project using the GUI. I get a file not found error:
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\myusername\\Anaconda3\\envs\\DEEPLABCUT\\lib\\site-packages\\deeplabcut\\modelzoo\\models.json
When I check the files, I see models.json there. Also the project folder is created containing the config.yaml file and other folders. The thing is I can't load it, and I never get past the create/load project screen in the GUI. 
Above in the thread title, it says may be fixed by #2170. As mentioned there, I added the line  "deeplabcut/modelzoo/models.json" to my DeepLabCut\setup.py file, but I still get the error.
Happy to provide additional info/screenshots...

**Comment by jeylau** [comment]
@kdhenderson, after adding the line, you'd need to reinstall deeplabcut too. If you have cloned the repo though, installing the code in editable mode with `pip install -e .` is probably the best option.

**Comment by kdhenderson** [comment]
Thanks @jeylau. This is a newbie question. Originally, I did clone the repo and created a conda env. If I want to install the code in editable mode, can I just cd into my DeepLabCut folder and pip install -e .
Essentially, I'm confused how my my pip install and the conda env relate to one another. I'm not sure if that question makes sense. Sorry I tried to google my way to the answer, but it seemed more advisable to try to ask rather than just start typing in commands...
